### ISSUE TITLE
Empty space before second comment

### ISSUE THREAD

**Issue by vreality64** [ardatan/graphql-toolkit#474]
Hello. I'm using `merge-graphql-schemas` which uses this schema merge. I checked the auto-generated SDL and found the following issues about comment symbol `#`.

**Source**
```graphql
type Some {
  # comment
  #  - first line
  #  - second line
  field: Int
}
```

**Merged - empty space after the second line**
```graphql
type Some {
  # comment
   #  - first line
   #  - second line
  field: Int
}
```

Empty space is look like bug. Is it intended?
https://github.com/ardatan/graphql-toolkit/blob/d2fbf6465fd4cda1e820eb9f1e17d153a9069e8d/packages/schema-merging/src/typedefs-mergers/comments.ts#L66-L68
In the test code, it was removed using the `stripWhitespaces` utility function. Here are [playground link](https://runkit.com/vreality64/graphql-toolkit-with-comments)

Also I have another question. Is there option for strip away `#` generation? I only want to keep not comment but description.

**Comment by ardatan** [comment]
We're working on a fix for indentation issue in #483. To have descriptions instead of comments, you can pass `commentDescriptions: false` to the options;
```ts
mergeTypes(types, { commentDescriptions: false });
```

**Comment by vreality64** [comment]
@ardatan 
> We're working on a fix for indentation issue in #483.

Glad to hear that. 👍

> `commentDescriptions: false` to the options

I tested `commentDescriptions` options, however return is not same type. ([playground](https://runkit.com/vreality64/comparison-commentdescription-options))

**with commentDescription**
```ts
mergeTypes(types, { commentDescriptions: true })
// -> SDL string
```

**without commentDescription**
```ts
mergeTypes(types, { commentDescriptions: false })
// -> Document Node
```

The reason is below code. However I'm not sure why this happen. In my opinion, it should be same due to option name.
https://github.com/ardatan/graphql-toolkit/blob/5071524af2d74318085c8f6c1390f96421da1f75/packages/schema-merging/src/typedefs-mergers/merge-typedefs.ts#L92-L96

**Comment by ardatan** [comment]
@vreality64 This is done on purpose, because `comments` are not the part of DocumentNode anymore so we need to return `string`.

**Comment by ardatan** [comment]
Available in v0.9.8 !!!
